src/context/Theme.tsx

```diff
@@ -288,6 +288,12 @@ export const jsonObjectPreview_value = {
 };
 
 // Styles
+// TODO (styling) - we use this styling on non-cards we should probably rename this
+//  and use in other places
+export const cardHeaderSx: SxProps<Theme> = {
+    fontSize: 16,
+    fontWeight: 500,
```

Comment:
The font weight should be 300.

Reply:
I'll update the weight - I am not moving around the component cause that is a major pain and that is why I skipped it when these were originally made. I'll mark it with a `todo`